tests/composite_pk/models/tenant.py

```diff
@@ -35,13 +36,14 @@ class Comment(models.Model):
         related_name="comments",
     )
     id = models.SmallIntegerField(unique=True, db_column="comment_id")
-    user_id = models.SmallIntegerField()
+    user_id = models.SmallIntegerField(null=True)
     user = models.ForeignObject(
         User,
         on_delete=models.CASCADE,
         from_fields=("tenant_id", "user_id"),
         to_fields=("tenant_id", "id"),
         related_name="comments",
+        null=True,
```

Comment:
Making both `null=True` allows to adequately test `pk_idx` implementation as @sarahboyce discovered.